scripts/cybershake/auto_submit.py

```diff
@@ -488,13 +516,14 @@ def main(args):
                 proc_type,
                 run_name,
                 root_folder,
-                queue_folder,
+                mgmt_queue_folder,
                 binary_mode=binary_mode,
                 hf_seed=hf_seed,
                 rand_reset=rand_reset,
                 extended_period=extended_period,
                 models=(lf_est_model, hf_est_model, bb_est_model, im_est_model),
             )
+            print()
```

Comment:
Why's this here?

Reply:
Don't forget to flush.

Reply:
Just a new line for the auto submit output to make it more readable.